### PR TITLE
Explain need for ZF to see true well-known data

### PR DESCRIPTION
**Pull request by richsalz**
Fixes: #2

### CODE REVIEW

**Comment by bemasc** [comment]
I would prefer a more specific assignment of responsibilities, like:

* The ZF MUST fetch it securely from the relevant origin.
* An HTTP Gateway MUST NOT modify or inject the resource.
* These rules can be changed by private arrangement.

**Comment by richsalz** [comment]
Fixup commit added that adds the specific assignments.

File: draft-ietf-tls-wkech.xml [hunk 1]
```diff
@@ -417,6 +417,24 @@ Client <-->|                    | |                    | facing
...
+	the well-known endpoint securely from the relevant origin, and any
+	HTTP gateways MUST NOT modify or inject their own version of the
+	resource.
+        Details about the nature of other arrangements are outside the scope
+        of this document, but possibilities include:
+      <list>
+	<t>A not-obvious DNS name for the true origin known to the ZF; this
+	is also common practice when a CDN or other intermediary is used in
+	  front of the origin.</t>
+	<t>Configurating the intermediary to not modify the well-known resource from
```
Comment (bemasc) [inline draft-ietf-tls-wkech.xml:432]:
```suggestion
	<t>Configuring any intermediaries to preserve the well-known resource from
```

File: draft-ietf-tls-wkech.xml [hunk 2]
```diff
@@ -417,6 +417,24 @@ Client <-->|                    | |                    | facing
...
+	  the origin.</t>
+	<t>Use a mechanism other than HTTP over TLS to fetch the data.</t>
```
Comment (bemasc) [inline draft-ietf-tls-wkech.xml:434]:
```suggestion
	<t>Using a mechanism other than HTTP over TLS to transfer the configuration.</t>
```

**Review by bemasc — APPROVED** [review]

**Pull request merged**